.github/workflows/pr-check.yaml

```diff
@@ -146,46 +159,93 @@ jobs:
           run_install: false
           package_json_file: ./podman-desktop/package.json
 
-      - name: Execute pnpm
+      - name: Install Podman Desktop dependencies
         working-directory: ./podman-desktop
         run: pnpm install
 
+      - name: Install SSO extension dependencies
```

Comment:
do we need to build sso ourselves? Isn't is possible to just `docker pull sso` and then create container and export, etc. ?

Reply:
I tried earlier that approach, but it had the same issue as building the crc extension that way (complaints that the image is built for linux). I think it's related to the default running containers being windows (not 100% sure), but after several unsuccessful tries to toggle that setting, I figured that building the extension locally like the other one would be just more straightforward.

Reply:
the complaining is may related to how you unpack the extension to set them on an spericif path? ....may if you add the feature on the testing framework to just pass the extension image url and directly install trought the UI?

Official releases for extensions are build only once and used across all platforms

Reply:
that is too late. when we open PD with locally prepared crc-extension it must have the dependency present (sso-extension) already otherwise the state of extension will be failed.